modules/ROOT/pages/release-notes.adoc

```diff
@@ -23,6 +23,16 @@ These credentials can be consulted and modified:
 * in the <BONITA>/server/conf/tomcat-users.xml file in a *Bonita Tomcat bundle*
 * using xref:bonita-docker-installation.adoc#_monitoring_username[specific Environment parameters] when starting a *Bonita Docker image*
 
+
+=== Dynamic authorization checking active by default (Subscription editions)
+
+xref:identity:rest-api-authorization.adoc#dynamic_authorization[Dynamic authorization checking] is now only available in the *Enterprise* editions, and is*active by default* for new installations. +
+Existing installations that update to 2022.1 stay *unchanged*, but we encourage customers to activate dynamic authorization checking, once they have benched that the performance cost is reasonable.
+
+Community users that update to 2022.1 simply lose the possibility to use dynamic authorization checking.
+Their potential previous configuration regarding dynamic authorizations is kept unchanged during the update procedure,
+to be used as a reference for custom usage, or ready to use for a potential upgrade to Enterprise edition.
```

Comment:
As a reader who do not know the subject I'm not sure to understand, is the previous configuration still working or not? 
I suggest to be a bit more direct, if the conf is kept but doesn't work anymore say it, even if it is a bit redundant with the sentence 'lose the possibility to use dynamic authorization checking'. And if the conf is still working but community users cannot create new ones then some sentences should be changed.

Reply:
The previous configuration is simply kept as is, but not taken into account anymore in Community edition.

@alachambre how do you suggest to rephrase it?

Reply:
Here is another suggestion:
"For Community users who update to 2022.1,  dynamic authorization checking does not work anymore. +
However, the configuration stays unchanged, even if it is not used anymore. +
If they want to reuse the configuration in a custom manner, or upgrade to Enterprise edition, the configuration will be back there and fully functional."

What do you think?

Reply:
looks good 👍